syntax-highlighting-code-block.php

```diff
@@ -222,3 +233,71 @@ function render_block( $attributes, $content ) {
 		);
 	}
 }
+
+/**
+ * Initialize admin settings.
+ */
+function admin_init() {
+	register_setting( 'syntax_highlighting', 'syntax_highlighting_options' );
+}
+add_action( 'admin_init', __NAMESPACE__ . '\admin_init' );
+
+/**
+ * Sanitize the given theme name to make sure it's a valid highlight.js theme.
+ *
+ * @param string $input Value of the dropdown.
+ *
+ * @return string The sanitized theme name
+ */
+function sanitize_theme_name( $input ) {
+	require_once __DIR__ . '/vendor/scrivo/highlight.php/HighlightUtilities/functions.php';
+
+	$themes = \HighlightUtilities\getAvailableStyleSheets();
+
+	return in_array( $input, $themes, true ) ? sanitize_text_field( $input ) : 'default';
+}
+
+/**
+ * Add plugin settings to Customizer.
+ *
+ * @param \WP_Customize_Manager $wp_customize The Customizer object.
+ */
+function customize_register( $wp_customize ) {
```

Comment:
I've added the short-circuit but will this behavior be documented somewhere? Otherwise, there's no way of knowing that this is the reason why the selector doesn't exist.

Reply:
Good question. Perhaps that would be good to document in the PhpDoc for the `syntax_highlighting_code_block_style` filter. Also, it can be mentioned in `readme.txt` when discussing the filter.

Reply:
Added to the PHPDoc and added a note to the `readme.txt`. Not sure if you'd prefer to word it differently or put more of an emphasis on the options.